admin_customers.php

```diff
@@ -913,10 +913,13 @@
 						$domainid = Database::lastInsertId();
 
 						// set ip <-> domain connection
-						$ins_stmt = Database::prepare("
-							INSERT INTO `".TABLE_DOMAINTOIP."` SET `id_domain` = :domainid, `id_ipandports` = :ipid"
-						);
-						Database::pexecute($ins_stmt, array('domainid' => $domainid, 'ipid' => Settings::Get('system.defaultip')));
+						$defaultips = explode(',', Settings::Get('system.defaultip'));
+						foreach ($defaultips as $defaultip) {
+							$ins_stmt = Database::prepare("
```

Comment:
The statement does not need to be prepared multiple times, once before the foreach would be enough